src/span.js

```diff
@@ -11,27 +11,37 @@
 // or implied. See the License for the specific language governing permissions and limitations under
 // the License.
 
-import * as constants from './constants.js';
-import SpanContext from './span_context.js';
-import * as opentracing from 'opentracing';
-import Utils from './util.js';
 import BaggageSetter from './baggage/baggage_setter';
+import * as constants from './constants';
+import * as opentracing from 'opentracing';
+import SpanContext from './span_context';
+import Tracer from './tracer';
+import Utils from './util';
 
 export default class Span {
-  _tracer: any;
+  static _baggageHeaderCache: {};
```

Comment:
Can we avoid moving code around in this diff? Or do it separately on master. This is a difficult change, I'd rather avoid additional refactoring noise.